webdriver-spec.html

```diff
@@ -5777,19 +5778,25 @@
  <li><p>Let <var>body</var> and <var>arguments</var>
   be <var>script arguments</var>’ data.
 
- <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
-  with arguments <var>body</var> and <var>arguments</var>.
+ <li><p>Let <var>promise</var> be a new Promise.
 
- <li><p>Wait for <a>execute a function body</a> to return
-  or for <a>session script timeout</a> milliseconds to expire,
-  whichever happens sooner.
+ <li><p>Run the following substeps <a>in parallel</a>:
+   <ol>
+     <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
```

Comment:
Either this, or the `execute a function body` section (GitHub won't let me add a comment to an unmodified line...) should be reworded to use `promise-calling`:

https://www.w3.org/2001/tag/doc/promises-guide#promise-calling